tests/test_docker_cat.py

```diff
@@ -0,0 +1,539 @@
+        for line in (expected_sensor, expected_import):
+            # Hint: if this test fails, the sensor for the tool or for the importation was not launched
+            assert line in analysis_output
+        # Wait for SonarQube to process the results
+        time.sleep(10)
+        # Check that the issue was added to the project
+        issues = requests.get(f"{cls.CAT_URL}/api/issues/search?componentKeys={project_key}",
+            auth=("admin", "admin")).json()['issues']
+        nb_issues = len([ issue for issue in issues if issue['rule'] == rule_violated ])
+        # Hint: an issue must be raised by the rule violated
+        assert nb_issues == 1
+        # Delete the project
+        requests.post(f"{cls.CAT_URL}/api/projects/delete",
+            auth=("admin", "admin"),
+            data={"project": project_key})
+        if activate_rule:
+            # Deactivate the rule in the Quality Profile
+            requests.post(f"{cls.CAT_URL}/api/qualityprofiles/deactivate_rule",
+                auth=("admin", "admin"),
+                data={
+                    "key": qp_key,
+                    "rule": rule_violated
+                })
+
+    @classmethod
+    def setup_class(cls):
+        """Set up the tests
+        Launch a container and wait for it to be up
+        """
+        docker_client = docker.from_env()
+        # Launch a CNES SonarQube container
+        if cls.RUN == "yes":
+            print(f"Launching lequal/sonarqube container (name={cls.CAT_CONTAINER_NAME})...")
+            docker_client.containers.run(cls._DOCKER_CAT_IMAGE,
+                name=cls.CAT_CONTAINER_NAME,
+                detach=True,
+                auto_remove=True,
+                environment={"ALLOWED_GROUPS": os.getgid()},
+                ports={9000: 9000},
+                volumes={
+                    f"{cls._PROJECT_ROOT_DIR}": {'bind': '/media/sf_Shared', 'mode': 'rw'},
+                })
+        else:
+            print(f"Using container {cls.CAT_CONTAINER_NAME}")
+        # Wait for the SonarQube server inside it to be set up
+        print(f"Waiting for {cls.CAT_CONTAINER_NAME} to be up...")
+        cls.wait_cat_ready(cls.CAT_CONTAINER_NAME)
+
+    @classmethod
+    def teardown_class(cls):
+        """Stop the container
+        """
+        if cls.RUN == "yes":
+            print(f"Stopping {cls.CAT_CONTAINER_NAME}...")
+            docker_client = docker.from_env()
+            docker_client.containers.get(cls.CAT_CONTAINER_NAME).stop()
+
+    def test_up(self):
+        """As a user, I want the server to be UP so that I can use it.
+        """
+        status = requests.get(f"{self.CAT_URL}/api/system/status",
+                    auth=("admin", "admin")).json()['status']
+        # Hint: if this test fails, the server might still be starting
+        assert status == "UP"
+
+    def test_check_plugins(self):
+        """As a SonarQube user, I want the plugins listed in the README
```

Comment:
Codacy found an issue: [1 blank line required between summary line and description (found 0) (D205)](https://app.codacy.com/gh/cnescatlab/docker-cat/pullRequest?prid=6253839)